app/views/projects/_no_shared_runners.html.haml

```diff
@@ -0,0 +1,5 @@
+.alert.alert-danger
+  %p
+    There are NO runners to build this project. Consider assigning this project to
+    %span.label.label-info a specific
```

Comment:
What does this look like in the UI? Does it make sense to have `a specific` inside the label or on the line before? Same with the specific runner partial.